data/human/free worlds side plots.txt

```diff
@@ -11,6 +11,89 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+
+
+mission "FW Convoy to Zug"
+	name "Convoy to Zug"
+	description "Escort these Freighters to Zug for an important project. Expect pirate raids."
+	source
+		government "Free Worlds"
+		near Zubeneschamali 2 4
+	destination "Zug"
+	to offer
+		has "event: war begins"
+		has "FW Syndicate Diplomacy 1D: done"
+		not "fw bloodsea done"
+	on offer
+		conversation
+			`While returning from a walk around the spaceport town of <origin>, you see the quartermaster of the spaceport inspecting the <ship>. He notices your arrival, and comes up to greet you.`
+			`	"You're Captain <first> <last>, yes? We're looking for a good ship that can escort these freighters to <planet>." He points towards five Freighters in Free Worlds colors, some of which have considerable hull damage and are being repaired on the landing pads opposite to your ship.`
+			`	"Their last escort was just taken out by pirates, and we need someone who is experienced enough to fend off or evade them. You must have a reputation, since some of our militia captains recognized you. This is for quite a significant project on <planet>, so you will be paid <payment>."`
+			choice
+				`	"What is the project? It seems to be a big enterprise."`
+				`	"That's a fair few Freighters, should I be expecting any more pirates?"`
+					goto pirates
+				`	"Sorry, I'm not planning on going that way."`
+					decline
+			`	"The Freighters are carrying construction materials and specialized shipbuilding equipment for the extension of Zug's New Port, which has been a long time coming. It would greatly help the Free Worlds' efforts. I suppose that is why there have been pirates trying to pounce on these ships, the equipment would be a rich prize for them."`
```

Comment:
```suggestion
			`	"The Freighters are carrying construction materials and specialized shipbuilding equipment for the extension of Zug's New Port, which has been a long time coming. It would greatly help the Free Worlds' efforts. I suppose that is why there have been pirates trying to pounce on these ships; the equipment would be a rich prize for them."`
```